packages/svelte/tests/runtime-legacy/samples/attribute-boolean-case-insensitive/main.svelte

```diff
@@ -1 +1 @@
-<input READONLY={!0} REQUIRED={!1}>
+<input READONLY={true} REQUIRED={false}>
```

Comment:
this test was updated because one of the PRs introduced a naive/broken check for boolean-ness (i.e. looking for a boolean literal to determine whether to add an attribute or not), and changing it to a non-literal guarded against regressing. probably work keeping

```suggestion
<input READONLY={!0} REQUIRED={!1}>
```